input/pagecontent/ch-mhd-1.md

```diff
@@ -40,147 +40,20 @@ The DocumentReference elements which are allowed to be updated are defined in An
-      "valueCoding": {
-        "system": "urn:oid:2.16.756.5.30.1.127.3.10.6",
-        "code": "HCP",
-        "display": "Healthcare professional"
-      }
-    }
-  ],
-  "masterIdentifier": {
-    "system": "urn:ietf:rfc:3986",
-    "value": "urn:oid:1.3.6.1.4.1.12559.11.13.2.1.2951"
-  },
-  "identifier": [
-    {
-      "system": "urn:ietf:rfc:3986",
-      "value": "urn:uuid:50383ae5-49e5-4dea-b0e6-660cb9e7b91f"
-    }
-  ],
-  "status": "current",
-  "type": {
-    "coding": [
-      {
-        "system": "http://snomed.info/sct",
-        "code": "721912009",
-        "display": "Medication summary document (record artifact)"
-      }
-    ]
-  },
-  "category": [
-    {
-      "coding": [
-        {
-          "system": "http://snomed.info/sct",
-          "code": "422735006",
-          "display": "Summary clinical document (record artifact)"
-        }
-      ]
-    }
-  ],
-  "subject": {
-    "identifier": {
-      "system": "urn:oid:2.16.756.5.30.1.127.3.10.3",
-      "value": "761337610411353650"
-    }
-  },
-  "description": "Changed Description for -MedicationCard",
-  "securityLabel": [
-    {
-      "coding": [
-        {
-          "system": "http://snomed.info/sct",
-          "code": "17621005",
-          "display": "Normal (qualifier value)"
-        }
-      ]
-    }
-  ],
-  "content": [
-    {
-      "attachment": {
-        "contentType": "text/xml",
-        "language": "de-CH",
-        "url": "http://example.com/xdsretrieve?uniqueId=urn:uuid:413eb0f7-aa72-4405-86a4-7793a23fcc27&repositoryUniqueId=2.999.756.42.1",
-        "creation": "2020-06-29T11:58:00+00:00"
-      },
-      "format": {
-        "system": "http://ihe.net/fhir/ihe.formatcode.fhir/CodeSystem/formatcode",
-        "code": "urn:ihe:pharm:pml:2013",
-        "display": "Pharmacy PML"
-      }
-    }
-  ],
-  "context": {
-    "facilityType": {
-      "coding": [
-        {
-          "system": "http://snomed.info/sct",
-          "code": "264358009",
-          "display": "General practice premises (environment)"
-        }
-      ]
-    },
-    "practiceSetting": {
-      "coding": [
-        {
-          "system": "http://snomed.info/sct",
-          "code": "394802001",
-          "display": "General medicine (qualifier value)"
-        }
-      ]
-    },
-    "sourcePatientInfo": {
-      "reference": "#pat"
-    }
-  }
-}
 ```
+{% fragment DocumentReference/DocRefPdf JSON %}
+
 
 
 ##### Expected Actions
 
 The Document Responder shall provide a [CapabilityStatement](CapabilityStatement-CH.MHD.DocumentResponder.html) with the capabilities interaction and indicate that update is available on the DocumentReference.
 
-The Document Responder shall process the Update Document Metadata Request message and return the updated DocumentReference.
+The Document Responder shall process the Update Document Metadata Request message and return the updated DocumentReferences.
```

Comment:
I would keep it singular: the `Update Document Metadata Request message` is a single UPDATE operation, and returns a single updated DocumentReference.
If it's a transaction, then you have multiple such messages, with multiples responses.